WordPress/Classes/ViewRelated/Reader/ReaderMenuViewController.swift

```diff
@@ -145,6 +256,63 @@ import WordPressShared
         cell.imageView?.image = menuItem.icon
     }
 
+
+    func configureActionCell(cell: UITableViewCell, atIndexPath indexPath: NSIndexPath) {
+        guard let menuItem = viewModel.menuItemAtIndexPath(indexPath) else {
+            return
+        }
+
+        WPStyleGuide.configureTableViewActionCell(cell)
+
+        let image = Gridicon.iconOfType(.AddOutline)
+        let imageView = UIImageView(image: image)
+        imageView.tintColor = WPStyleGuide.wordPressBlue()
+        cell.accessoryView = imageView
```

Comment:
Minor potential optimization here, the `accessoryView` may already be an `UIImageView` from a previous configure call. So we could technically save a view creation here and just set the image on the existing `UIImageView`.